src/Swagger/Serializer/DocumentationNormalizer.php

```diff
@@ -163,6 +162,8 @@ public function __construct(ResourceMetadataFactoryInterface $resourceMetadataFa
         $this->paginationClientEnabled = $paginationClientEnabled;
         $this->paginationClientEnabledParameterName = $paginationClientEnabledParameterName;
 
+        $this->defaultContext[self::SPEC_VERSION] = (int) ($versions[0] ?? '2');
```

Comment:
Same here (what about the empty string).

Reply:
Nah, this one is coming from the configuration. It can't be an empty string there.
But.
This Normalizer comes from the api.xml file.
I had to handle the emptiness only because this service is not part of the swagger.xml file. Is it something we should change?